src/ducks/stripe.duck.js

```diff
@@ -549,7 +552,8 @@ export const handleCardPayment = params => dispatch => {
         return Promise.reject(response);
       } else {
         dispatch(handleCardPaymentSuccess(response));
-        return response;
+        const values = { ...response, id };
+        return values;
```

Comment:
How about just `return { ...response, orderId };`?
(My suggestion in previous comment was to rename "id" to "orderId" - but it can be "transactionId" too)